java/impl/aws/src/main/java/co/worklytics/psoxy/aws/AwsModule.java

```diff
@@ -66,6 +67,7 @@ static SsmClient ssmClient(AwsEnvironment awsEnvironment) {
                 .build();
     }
 
+    //NOTE: also used in SecretsManager case for simplicity
     static String asParameterStoreNamespace(String functionName) {
```

Comment:
rename to a more general `asSecretNamespace` ?

Reply:
Yeah, now is not only used for ParameterStore